venv/CalculationClass.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+
+
+#Query Analysis Calculation Class
+class Calculations():
+
+    def __init__(self,  dataframe,query):
```

Comment:
```py
    def __init__(self, dataframe, query):
```
Formatting (spacing) issues. 

Use a formatter like [black](https://github.com/psf/black) to handle formatting rather than doing it manually.